po/ru.po

```diff
@@ -990,49 +990,3 @@ msgid ""
 msgstr ""
 "Чтобы включить или выключить синхронизацию времени по сети, необходимо "
 "пройти аутентификацию."
-
-#~ msgid "Authentication is required to start '$(unit)'."
-#~ msgstr "Чтобы запустить «$(unit)», необходимо пройти аутентификацию."
-
-#~ msgid "Authentication is required to stop '$(unit)'."
-#~ msgstr "Чтобы остановить «$(unit)», необходимо пройти аутентификацию."
-
-#~ msgid "Authentication is required to reload '$(unit)'."
-#~ msgstr ""
-#~ "Чтобы заставить «$(unit)» перечитать конфигурацию, необходимо пройти "
-#~ "аутентификацию."
-
-#~ msgid "Authentication is required to restart '$(unit)'."
-#~ msgstr "Чтобы перезапустить «$(unit)», необходимо пройти аутентификацию."
-
-#~ msgid ""
-#~ "Authentication is required to send a UNIX signal to the processes of "
-#~ "'$(unit)'."
-#~ msgstr ""
-#~ "Чтобы отправить сигнал UNIX процессам юнита «$(unit)», необходимо пройти "
-#~ "аутентификацию."
-
-#~ msgid ""
-#~ "Authentication is required to reset the \"failed\" state of '$(unit)'."
-#~ msgstr ""
-#~ "Чтобы сбросить состояние «failed» у юнита «$(unit)», необходимо пройти "
-#~ "аутентификацию."
-
-#~ msgid "Authentication is required to set properties on '$(unit)'."
-#~ msgstr ""
-#~ "Чтобы изменить параметры юнита «$(unit)», необходимо пройти "
-#~ "аутентификацию."
-
-#~ msgid ""
-#~ "Authentication is required to delete files and directories associated "
-#~ "with '$(unit)'."
-#~ msgstr ""
-#~ "Чтобы удалить файлы и директории, относящиеся к юниту «$(unit)», "
-#~ "необходимо пройти аутентификацию."
-
-#~ msgid ""
-#~ "Authentication is required to freeze or thaw the processes of '$(unit)' "
-#~ "unit."
-#~ msgstr ""
-#~ "Чтобы отправить сигнал заморозки или разморозки процессам юнита "
-#~ "«$(unit)», необходимо пройти аутентификацию."
```

Comment:
No no, please do not drop them.
Please read https://github.com/systemd/systemd/blob/main/docs/TRANSLATORS.md.
Especially, use the following:
```
ninja -C build/ systemd-update-po
```

Reply:
> I updated the .po file and these strings were out of date

I do not observe that for ru.po with `ninja systemd-update-po`. Does your git repository is uptodate??

Reply:
Isn't this .pot relevant? https://l10n.gnome.org/POT/systemd.main/systemd.main.pot
 I used it